guides/source/action_view_helpers.md

```diff
@@ -507,166 +717,328 @@ number_with_precision(111.2345)               # => 111.235
 
-Strips all HTML tags from the html, including comments.
-This functionality is powered by the rails-html-sanitizer gem.
+Strips all HTML tags from the HTML, including comments.
 
 ```ruby
 strip_tags("Strip <i>these</i> tags!")
 # => Strip these tags!
+
+strip_tags("<b>Bold</b> no more! <a href='more.html'>See more</a>")
+# => Bold no more! See more
 ```
 
+NOTE: The output may still contain unescaped `<`, `>`, `&` characters and
+confuse browsers.
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags)
+for more information.
+
+### TextHelper
+
+Provides a set of methods for filtering, formatting and transforming strings.
+
+#### excerpt
+
+Given a `text` and a `phrase`, `excerpt` searches for and extracts the first
+occurrence of the `phrase`, plus the requested surrounding text determined by
+a `radius`. An omission marker is prepended/appended if the start/end of the
+result does not coincide with the start/end of the text.
+
 ```ruby
-strip_tags("<b>Bold</b> no more!  <a href='more.html'>See more</a>")
-# => Bold no more!  See more
+excerpt('This is a very beautiful morning', 'very', separator: ' ', radius: 1)
+# => ...a very beautiful...
+
+excerpt('This is also an example', 'an', radius: 8, omission: '<chop> ')
+#=> <chop> is also an example
 ```
 
-NB: The output may still contain unescaped '<', '>', '&' characters and confuse browsers.
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-excerpt)
+for more information.
 
-See the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags) for more information.
+#### pluralize
 
-### UrlHelper
+Returns the singular or plural form of a word based on the value of a number.
 
-Provides methods to make links and get URLs that depend on the routing subsystem.
+```ruby
+pluralize(1, 'person') # => 1 person
+pluralize(2, 'person') # => 2 people
+pluralize(3, 'person', plural: 'users') # => 3 users
+```
 
-#### url_for
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize)
+for more information.
 
-Returns the URL for the set of `options` provided.
+#### truncate
 
-##### Examples
+Truncates a given `text` to a given `length`. If the text is truncated, an omission
+marker will be appended to the result for a total length not exceeding `length`.
 
 ```ruby
-url_for @profile
-# => /profiles/1
+truncate("Once upon a time in a world far far away")
+# => "Once upon a time in a world..."
 
-url_for [ @hotel, @booking, page: 2, line: 3 ]
-# => /hotels/1/bookings/1?line=3&page=2
+truncate("Once upon a time in a world far far away", length: 17)
+# => "Once upon a ti..."
 
-url_for @post # given a composite primary key [:blog_id, :id]
-# => /posts/1_2
+truncate("Once upon a time in a world far far away", length: 17, separator: ' ')
+# => "Once upon a..."
```

Comment:
Maybe using an example with another separator would make this example more clear?
```suggestion
truncate("one-two-three-four", length: 17, separator: '-')
# => "one-two-three..."
```

Reply:
I love this, it's more clear thank you!